src/containers/my-resources/lessons-container/LessonsContainer.tsx

```diff
@@ -2,8 +2,9 @@ import SearchIcon from '@mui/icons-material/Search'
 import Box from '@mui/material/Box'
 import { ChangeEvent, useCallback, useRef, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { useNavigate } from 'react-router-dom'
+import { Link } from 'react-router-dom'
 
+import AddIcon from '@mui/icons-material/Add'
```

Comment:
import from library mui has to be together with imports of other libraries

Reply:
fixed